lighthouse-core/audits/seo/link-text.js

```diff
@@ -124,7 +124,17 @@ class LinkText extends Audit {
           return false;
         }
 
-        return BLOCKLIST.has(link.text.trim().toLowerCase());
+        const normalized = link.text.trim().toLowerCase();
+        const isJustOneCodePoint = [...normalized].length === 1;
+        const codePoint = isJustOneCodePoint && normalized.charCodeAt(0);
+        // https://www.w3schools.com/charsets/ref_utf_symbols.asp
+        const isUnicodeSymbol =
+          codePoint >= parseInt('2600', 16) && codePoint <= parseInt('26FF', 16);
+        const isFontAwesome =
+          codePoint >= parseInt('f000', 16) && codePoint <= parseInt('f2e0', 16);
+        const isMeaningfulIcon = isUnicodeSymbol || isFontAwesome;
```

Comment:
I'd personally prefer not flagging unicode symbols to start (which do have some shared semantic meaning, seems plausible that search could use that meaning until we hear otherwise, e.g. telephone for a telephone number link 🤷 ).

The private use area (which Font Awesome and many icon fonts use) is explicitly *not* shared though.
```suggestion
        // Private use area code points do not have shared meaning. Commonly used by icon fonts like Font Awesome.
        // @see http://www.unicode.org/faq/private_use.html
        const isPrivateUseArea =
          parseInt('E000', 16) <= codePoint && codePoint <= parseInt('F8FF', 16);
```